file2

update python fucntion with merge conflict
